src/styles/blocks/page.scss

```diff
@@ -0,0 +1,56 @@
+.page {
+  font-family: "Poppins", sans-serif;
+  background: #1d1d1d;
+  &__body {
```

Comment:
on very large screens the site stretches too much, maybe you should add max-width?

![bigscreen](https://user-images.githubusercontent.com/89162624/209443108-efa42ca7-ac39-443c-b5bc-08fbee21ced4.png)